flask_resty/view.py

```diff
@@ -288,12 +284,12 @@ def deserializer(self):
         """
         return self.schema
 
-    def format_validation_error(self, error):
-        """Convert a marshmallow validation error to a serializable form.
+    def format_validation_error(self, message, path):
```

Comment:
suggestion (non-blocking): add typing, since this is public API?

```suggestion
    def format_validation_error(self, message: str, path: typing.Sequence[str]):
```

Reply:
let me just add some types to the docstring for now... i don't want to have just a single function with a type signature sitting on its own